<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="Caption_Label">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Выбран алгоритм: Зеркало</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Вход</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="inpTE"/>
        </item>
        <item>
         <widget class="QPushButton" name="OpenButton">
          <property name="text">
           <string>Открыть файл</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Ключ</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="keyLE"/>
        </item>
        <item>
         <widget class="QPushButton" name="CryptButton">
          <property name="text">
           <string>Шифровать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="decryptButton">
          <property name="text">
           <string>Дешифровать</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Выход</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="outTE">
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SaveButton">
          <property name="text">
           <string>Сохранить в файл</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>505</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Алгоритм шифрования</string>
    </property>
    <addaction name="choose_mirror"/>
    <addaction name="choose_skitala"/>
    <addaction name="choose_cipher_of_Caesar"/>
    <addaction name="Choose_Vigner"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Тест</string>
    </property>
    <addaction name="test"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="choose_mirror">
   <property name="text">
    <string>&quot;Королевство кривых зеркал&quot;</string>
   </property>
  </action>
  <action name="choose_skitala">
   <property name="text">
    <string>Скитала</string>
   </property>
  </action>
  <action name="choose_cipher_of_Caesar">
   <property name="text">
    <string>Простая замена(Шифр Цезаря)</string>
   </property>
  </action>
  <action name="test">
   <property name="text">
    <string>Тест</string>
   </property>
  </action>
  <action name="Choose_Vigner">
   <property name="text">
    <string>Шифр Вижнера</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
